acm/deploy.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+set -e
+
+kubectl create namespace ${MCE_NS} --dry-run=client -o json | kubectl apply -f -
+
+set +e
+# Ensure smooth upgrade from mce 2.7.0 to 2.8.1
+helm uninstall --ignore-not-found \
+    clc-state-metrics \
+    --namespace ${MCE_NS}
+set -e
+
+phase=$(kubectl -n multicluster-engine get mce multiclusterengine -o json | jq -r '.status.phase')
+
+if [ "${phase}" = "Paused" ]; then
+    echo "MCE is already paused, skipping deploy"
+    exit 0
+fi
```

Comment:
how will unpause work?

Reply:
Manually remove the annotation

Reply:
maybe we can enhance the condition for `exit 0` and check for MCE_PAUSE_RECONCILIATION == true as well.

* in case MCE does not exist yet, `phase` will be empty, so not matching `Pause` and will proceed with installing MCE (potentially pausing it at the end of the script and scaling it down in the next step
* in case MCE exists and is paused + MCE_PAUSE_RECONCILIATION indicates it should be paused, we can exit since there is nothing to do here
* in case MCE exists and is paused but MCE_PAUSE_RECONCILIATION indicates it should not be paused, we proceed with the MCE deployment (which scales it up again) - at the end of the script, the pause annotation is also set to false